examples/Vendored-XCFramework12-Example/BananaLib/build/watchOS.xcarchive/dSYMs/BananaLib.framework.dSYM/Contents/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does the `build` folder need to be checked in? particularly the xcarchives, it doesn't look like they are used or compared against the generated output

Reply:
@amorde you're right -- the xcarchives do not need to be checked in. I'm making a tweak to the build scripts to push the generated archives down a level into `build/DerivedData/` which will get them ignored by the global gitignore.